TenureInformationApi.Tests/SnsEventVerifier.cs

```diff
@@ -0,0 +1,128 @@
+using Amazon.SimpleNotificationService;
+using Amazon.SQS;
+using Amazon.SQS.Model;
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace TenureInformationApi.Tests
+{
+    // TODO - move somewhere common?
+
+    /// <summary>
+    /// Helper class used to verify that the correct Sns event gets raised during an E2E test
+    /// </summary>
+    /// <typeparam name="T">The event class used to create the event.</typeparam>
+    public class SnsEventVerifier<T> : IDisposable where T : class
```

Comment:
one for `Hackney.Core.Sns.Tests` ?

Reply:
Certainly one for a common package I agree.